devtools/create_project/create_project.cpp

```diff
@@ -1082,6 +1095,17 @@ const Tool s_tools[] = {
 	{ "create_translations", true},
 	{ "qtable",              true}
 };
+
+const MSVCVersion s_msvc[] = {
+//    Ver    Name                     Solution                     Project    Toolset    LLVM
+	{  9,    "Visual Studio 2008",    "10.00",          "2008",     "4.0",     "v90",    "LLVM-vs2008" },
+	{ 10,    "Visual Studio 2010",    "11.00",          "2010",     "4.0",    "v100",    "LLVM-vs2010" },
+	{ 11,    "Visual Studio 2012",    "11.00",          "2012",     "4.0",    "v110",    "LLVM-vs2012" },
+	{ 12,    "Visual Studio 2013",    "12.00",          "2013",    "12.0",    "v120",    "LLVM-vs2013" },
+	{ 14,    "Visual Studio 2015",    "12.00",            "14",    "14.0",    "v140",    "LLVM-vs2014" },
+	{ 15,    "Visual Studio 2017",    "12.00",            "15",    "15.0",    "v141",    "llvm"        },
+	{ 16,    "Visual Studio 2019",    "12.00",    "Version 16",    "16.0",    "v142",    "llvm"        }
```

Comment:
Just double checking, they moved from xx to Version xx?

Reply:
Yup. For reference, here's the whole thing:
```
Microsoft Visual Studio Solution File, Format Version 12.00
# Visual Studio Version 16
```
I don't think this information is actually *necessary*, but why fix what isn't broken.